hooks/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Is this script made for CI pipeline?

Reply:
It's picked up by Docker Hub automated build, see [this page](https://docs.docker.com/docker-hub/builds/advanced/) about it. But it won't be picked up by Circle CI.

Reply:
Oh, TIL, thanks.